app/form-components/FormTemplate.tsx

```diff
@@ -36,6 +36,26 @@ const HeaderContainer = styled.div`
   min-height: 150px;
 `;
 
+const isBceidBoth = (idp: string) => idp === 'bceidboth';
+const hasIdir = (idp: string) => ['idir', 'azureidir'].includes(idp);
+const hasBceidRegular = (idp: string) => ['bceidbasic', 'bceidbusiness'].includes(idp);
+const noBceidBoth = (idp: string) => hasIdir(idp) || idp !== 'bceidboth';
+const noBceidRegular = (idp: string) => hasIdir(idp) || !hasBceidRegular(idp);
+
+const filterIdps = (currentIdps: string[], updatedIdps: string[]) => {
+  const idpAdded = currentIdps.length < updatedIdps.length;
```

Comment:
Is it possible for an edge case to happen where an idp is added and a different one removed at the same time? That would prevent this from being triggered, but I'm not sure if that's even possible

Reply:
right, we don't need to worry about that edge case that this logic is always triggered one at a time.

Reply:
awesome, thanks